aws-athena/ctas/location-census_acs5.sql

```diff
@@ -93,19 +94,19 @@ WITH (
 
     SELECT
         pcl.pin10,
-        ayf.census_acs5_congressional_district_geoid,
-        ayf.census_acs5_county_subdivision_geoid,
-        ayf.census_acs5_place_geoid,
-        ayf.census_acs5_puma_geoid,
-        ayf.census_acs5_school_district_elementary_geoid,
-        ayf.census_acs5_school_district_secondary_geoid,
-        ayf.census_acs5_school_district_unified_geoid,
-        ayf.census_acs5_state_representative_geoid,
-        ayf.census_acs5_state_senate_geoid,
-        ayf.census_acs5_tract_geoid,
+        dj.census_acs5_congressional_district_geoid,
+        dj.census_acs5_county_subdivision_geoid,
+        dj.census_acs5_place_geoid,
+        dj.census_acs5_puma_geoid,
+        dj.census_acs5_school_district_elementary_geoid,
+        dj.census_acs5_school_district_secondary_geoid,
+        dj.census_acs5_school_district_unified_geoid,
+        dj.census_acs5_state_representative_geoid,
+        dj.census_acs5_state_senate_geoid,
+        dj.census_acs5_tract_geoid,
```

Comment:
I'm pretty sure this was a mistake in the original table definition; as the query was written initially, Athena complained that no such attributes could be resolved from the `acs5_year_fill` subquery (aliased to `ayf` here). This fix seemed to correct to me, but I could be wrong!